docs/guide/Alloy_Framework/Alloy_Getting_Started.md

```diff
@@ -48,7 +48,7 @@ A new skeleton Alloy project will be generated. If you are familiar with classic
 To create a new Alloy project, use the Appcelerator CLI to create a new Alloy project. Run the following command in a terminal:
 
 ```bash
-appc new -t titanium
+ti create -t app
```

Comment:
Isn't `app` the default? Do we need the `-t app`?

Reply:
50/50, you got the choice, but `app` is set to default